src/lib/components/table/proposals/Proposer.tsx

```diff
@@ -4,14 +4,21 @@ import { useGetAddressType } from "lib/app-provider";
 import { ExplorerLink } from "lib/components/ExplorerLink";
 import type { Addr, Option } from "lib/types";
 
-export const Proposer = ({ proposer }: { proposer: Option<Addr> }) => {
+export const Proposer = ({
+  proposer,
+  amptrackSection,
+}: {
+  proposer: Option<Addr>;
+  amptrackSection?: string;
+}) => {
   const getAddressType = useGetAddressType();
 
   return proposer ? (
     <ExplorerLink
       type={getAddressType(proposer)}
       value={proposer}
       showCopyOnHover
+      ampCopierSection={amptrackSection}
```

Comment:
just hardcode proposal list if its only used in that page

Reply:
This proposer is also use in account details page